### ISSUE TITLE
Creating a new file in the file browser closes two open editor tabs

### ISSUE THREAD

**Issue by jimray** [alloy/macvim#56]
When you create a new file in the file browser (by right-clicking and selecting "New File" for instance) it deletes two open editor tabs. To replicate:
1. Open at least two tabs. These can be new tabs (via Command-T) or just files
2. Create a new file by right-clicking in the browser pane and selecting "New File"
3. Name the file (e.g. "foo")
4. The two most recently opened tabs are closed and the new file ("foo") is opened

**Comment by kelan** [comment]
This also happens if you rename or delete a file from the File Browser (using the contextual menu).  It doesn't seem to matter if it's the file that's visible in the current tab or not.

**Comment by alloy** [comment]
Yeah, I’ve been wanting to look into this for a while now, but my spare time is mostly spent at cocoapods.org atm. If anyone could look into this, even if it's just some investigation, that would be much appreciated.

**Comment by jimray** [comment]
As a follow up, it seems like deleting files in the sidebar exhibits a similar behavior, in that it closes the currently open tab, even if it's not the tab being deleted. To replicate:
1. Open a directory with multiple files that show up in the file browser
2. Open at least 2 tabs
3. Delete one of the files in the file browser
4. The most recently opened tab closes

My apologies, I wouldn't even know where to begin with mucking through the source code on fixing this, happy to help debug, though!

DAMN! I somehow skipped Kelan's comment on this. Sorry for the dupe!

**Comment by alloy** [comment]
Thanks, any info is helpful :)
